test/tst.protocol_encoder.js

```diff
@@ -255,8 +298,8 @@ function runTestCase(testcase)
 		 */
 		mod_assertplus.ok(Buffer.isBuffer(outbuf));
 		mod_assertplus.ok(outbuf.length > mod_protocol.FP_HEADER_SZ);
-		mod_assertplus.equal(1,
-		    outbuf.readUInt8(mod_protocol.FP_OFF_VERSION));
+//		mod_assertplus.equal(mod_protocol.FP_VERSION_CURRENT,
+//		    outbuf.readUInt8(mod_protocol.FP_OFF_VERSION));
```

Comment:
Forgotten debugging changes here?

Reply:
Oh yeah, those just need to be deleted. The version checks moved into the individual test cases now.